tests/Browser/ProjectTest.php

```diff
@@ -53,19 +61,28 @@ public function testProjectCreateAuth()
         });
     }*/
 
+    public function testAllProjectsShown() {
+        $this->browse(function (Browser $browser) {
+            $amountOfProjects = Project::all()->count();
+            $browser->visitRoute('projects');
+            $cards = $browser->elements('.project-card');
+            self::assertCount($amountOfProjects, $cards);
+        });
+    }
+
     public function testProjectDeleteNoAuth(){
         $this->browse(function (Browser $browser) {
             $browser->visitRoute('panelproject.remove', ['id' => 2])
                 ->assertRouteIs('login');
         });
     }
 
-    public function testProjectDeleteAuth() {
+    /*public function testProjectDeleteAuth() {
         $this->browse(function (Browser $browser) {
             $browser->loginAs(User::find(1))
                 ->visitRoute('panelproject.remove', ['id' => 2]);
         });
-    }
+    }*/
```

Comment:
Is this necessary?

Reply:
Could technically be removed, is a useless test. But since we are going to do code cleanup and I was not sure if it had some kind of intention I left it.